packages/table/src/table.tsx

```diff
@@ -1004,6 +1092,53 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         this.handleSelection([]);
     }
 
+    private handleFocusMove = (e: KeyboardEvent, direction: "up" | "down" | "left" | "right") => {
```

Comment:
Hmm, should we create a `RelativeDirection` enum or something? May not be necessary, but feels 1% cleaner.